ttnn/cpp/ttnn/operations/moreh/moreh_adam/device/moreh_adam_program_factory.cpp

```diff
@@ -277,6 +298,8 @@ void MorehAdamOperation::ProgramFactory::override_runtime_arguments(
             if (max_exp_avg_sq_in_buffer != nullptr) {
                 runtime_args[4] = max_exp_avg_sq_in_buffer->address();
             }
+            runtime_args[5] = f2u_lr.u;
+            runtime_args[10] = operation_attributes.step;
```

Comment:
just curious, why step index is 10 in `runtime_args`?

Reply:
I wish I have a hint of an idea what went through my colleague's minds when they wrote these ops